site/zenodo_rdm/migrator/stream_definition.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Migrator stream definitions."""
+
+from invenio_rdm_migrator.extract import Extract
+from invenio_rdm_migrator.streams import StreamDefinition
+from invenio_rdm_migrator.transform import Transform
+
+
+class ZenodoDirectLoadStreamDefition(StreamDefinition):
+    """Zenodo to RDM direct loading class i.e no ET steps."""
+
+    class NoExtract(Extract):
+        """Dummy class to satisfy the `invenio_rdm_mirgator.Extract` interface."""
+
+        def run(self):
+            """Yield one element at a time."""
+            # yield a dummy value
+            yield
+
+    class NoTransform(Transform):
+        """Dummy class to satisfy the `invenio_rdm_mirgator.Transform` interface."""
+
+        def _transform(self, entry):
+            """Transform entry."""
+            yield entry
+
+    def __init__(self, name, load_cls):
+        """Constructor."""
+        super(ZenodoDirectLoadStreamDefition, self).__init__(
+            name=name,
+            extract_cls=self.NoExtract,
```

Comment:
```suggestion
            extract_cls=self.NoExtract,
```

shelve: alternative would me to not pass anything (and default to this), but this requires changes to the base class. Definitely something that we could move in `rdm-migrator`.